src/components/Home.tsx

```diff
@@ -46,14 +46,13 @@ const ProjectPageLink = styled(NavLink)<TypographyProps & FlexboxProps>`
   justify-content: flex-start;
   ${flexbox}
 `;
+
 const AlphaLogo = styled.img<LayoutProps & SpaceProps>`
   ${layout};
   ${space};
   max-width: 40vw;
   max-height: 60vh;
-  transition: transform 0.5s;
-  &:hover {
-    transform: scale(1.01);
+  transition: 1.5s;
```

Comment:
this transition has no effect

Reply:
quite right

Reply:
with one exception.. when you play around with different screen sizes

the transition accentuates the effect of growth and shrink, 1.5 is roughly similar to the speed the ball rotates which I thought looked quite unusual,  in a nice way

Reply:
i think resizing our browser window is a false check that we developers fall into, a user rarely resizes their window

Reply:
sure, I can understand that. Are you willing to make the exception in the context of this project given its quirky nature?